package.json

```diff
@@ -40,13 +40,13 @@
         "bluebird": "3.7.2",
         "cli-progress": "3.10.0",
         "command-exists": "1.2.9",
-        "conf": "10.1.1",
```

Comment:
pls dont do that without dedicated PRs per update to track the update

Reply:
reverted